client/src/components/graph/graph.js

```diff
@@ -298,7 +319,7 @@ class Graph extends React.Component {
       (responsive.height && responsive.width && !centroidSVG)
     ) {
       // First time for centroid or label change
-      createCentroidSVG();
+      // createCentroidSVG();
```

Comment:
please remove unused code

Reply:
Was leaving this in to allow for a quick reinstantiation of the feature. Removed the comments and added a clarifying comment above